src/omv/ov5640.c

```diff
@@ -612,7 +612,8 @@ static int set_framesize(sensor_t *sensor, framesize_t framesize)
             break;
         case PIXFORMAT_GRAYSCALE:
         case PIXFORMAT_YUV422:
-            pll = (w > 2048) ? 0x50 : 0x64; // 32 MHz vs 40 MHz
```

Comment:
Please verify that RGB565/BAYER/GRAYSCALE all work for 0x64 and then just merge those into one case and remove the dead code. Note that there's another place in the code int eh set_pixformat method that you must change too. Ignore JPEG for now. That needs to be fixed in another way.

Reply:
Will do...

Reply:
Verified; I just pushed the change which sets the speed to 0x64 for everything. All pixel formats are stable at the highest resolution.